package.json

```diff
@@ -17,10 +17,10 @@
     "scripts": {
         "start": "electron-forge start",
         "package": "electron-forge package",
-        "make": "electron-forge make",
-        "make:mac": "electron-forge make -p darwin",
-        "make:linux": "electron-forge make -p linux",
-        "make:windows": "electron-forge make -p win32",
+        "make": "VITE_BASE_HREF=./ electron-forge make",
```

Comment:
if you refresh the page in a build (cmd + shift+ r), does the page load?
Can't test, the build is failing for me

Reply:
It seems the packaged electron doesn't support the browser react router. Will change it to use the MemoryRouter